src/Ratio.tsx

```diff
@@ -38,7 +38,7 @@ const defaultProps = {
 
 function toPercent(num: number): string {
   if (num <= 0 || num > 100) return '100%';
-  if (num < 1) return `${num * 100}%`;
+  if (num < 5) return `${num * 100}%`;
```

Comment:
It's better to remove the upper limit of 100 in the line above so users can specify 200 as the value

Reply:
Ah of course - Thanks!
I've committed your suggestion with a change to tests to allow the taller aspect ratio.